### PR TITLE
Nexus skill

### PR DESCRIPTION
**Pull request by jimmytyyang**

### CODE REVIEW

File: spot_rl_experiments/spot_rl/utils/img_publishers.py [hunk 1]
```diff
@@ -666,10 +666,12 @@ def inference(self, hand_rgb, timestamp, stopwatch):
         # We decide to hardcode these classes first as this will be more robust
         # and gives us the way to control the detection
         # multi_classes = [["ball", "cup", "table", "cabinet", "chair", "sofa"]]
-        multi_classes = [["glass bottle"]]
-        self.owlvit.update_label(multi_classes)
+        multi_classes = rospy.get_param("/multi_class_object_target").split(",")
+        self.owlvit.update_label([multi_classes])
```
Comment (KavitShah1998) [inline spot_rl_experiments/spot_rl/utils/img_publishers.py:670]: Here `multi_classes` is already a list right (as we called .split() ) ?
Then we can just pass it to `owlvit.update_label` ?

**Comment by KavitShah1998** [comment]
https://github.com/facebookresearch/spot-sim2real/pull/214/files#diff-a5b8eac17230285c05f88335ad5821aa16beb1580516a946bafdf0e2475ec436R713-R725

@jimmytyyang 
1. Do we need both `OWLVITModel` and `OWLVITModelMultiClasses` both initialized together?
2. How to handle ros img viz ? Should this visualizer show OWLVITModelMultiClasses or OWLVITModel ?

File: spot_rl_experiments/spot_rl/utils/img_publishers.py (hunk 1 above)
Comment (jimmytyyang) [reply spot_rl_experiments/spot_rl/utils/img_publishers.py:670]: owlvit model needs to be [["cup"], ["apple"]] format, which is a bit weird.

**Comment by jimmytyyang** [comment]
> https://github.com/facebookresearch/spot-sim2real/pull/214/files#diff-a5b8eac17230285c05f88335ad5821aa16beb1580516a946bafdf0e2475ec436R713-R725
> 
> @jimmytyyang
> 
> 1. Do we need both `OWLVITModel` and `OWLVITModelMultiClasses` both initialized together?
> 2. How to handle ros img viz ? Should this visualizer show OWLVITModelMultiClasses or OWLVITModel ?

Hi Kavit! Thanks for the feedback!

I want to have both to be initialized so that they do not interfere with each other. For the ros img viz, I am thinking about we only show the original OWLVITModel as OWLVITModelMultiClasses is solely used for Nexus app

**Review by KavitShah1998 — APPROVED** [review]

**Comment by KavitShah1998** [comment]
Let's merge this PR after we cleanup the current main.

**Pull request merged**